.github/workflows/tag-changes.yml

```diff
@@ -12,12 +12,6 @@ jobs:
         uses: actions/checkout@v2
         with:
           fetch-depth: 0
-      - name: Generate Token
```

Comment:
I copied this from the hermit-ij-plugin, but reading the [docs](https://github.com/mathieudutour/github-tag-action?tab=readme-ov-file#-inputs) on `mathieudutour/github-tag-action`, it seems like we might just need `secrets.GITHUB_TOKEN` which [should be auto assigned to each workflow run.](https://docs.github.com/en/actions/security-for-github-actions/security-guides/automatic-token-authentication#about-the-github_token-secret)